scripts/configure-dh.sh

```diff
@@ -331,6 +368,7 @@ for f in $BASE_DIR/dynamic-plugins/*.yaml; do
 
     echo "OK"
 done
+rm -f $BASE_DIR/dynamic-plugins/lightspeed-plugins.yaml
```

Comment:
Same as comment re: keeping them in the `/dynamic-plugins` dir, if we move them we won't need this line I believe

Reply:
This line could be removed if going with my suggestions. Otherwise if keeping the file copy approach this can be kept in.